{"name": "Send event with wrong data type for videoTime", "status": "failed", "statusDetails": {"message": "AssertionError: ❗ Expected validation error for bad 'videoTime', but got 200\nassert 200 >= 400\n +  where 200 = <Response [200]>.status_code", "trace": "self = <test_api_event.TestApiEvent object at 0x0000022C54FB5850>\n\n    @allure.title(\"Send event with wrong data type for videoTime\")\n    def test_post_invalid_video_time(self):\n        payload = {\n            \"userId\": \"user-123\",\n            \"type\": \"pause\",\n            \"videoTime\": \"not-a-number\",\n            \"timestamp\": datetime.utcnow().isoformat() + \"Z\"\n        }\n    \n        response = requests.post(BASE_URL, json=payload)\n        logger.info(f\"Invalid 'videoTime' type - Status: {response.status_code}\")\n>       assert response.status_code >= 400, (\n            \"❗ Expected validation error for bad 'videoTime', but got \"\n            f\"{response.status_code}\"\n        )\nE       AssertionError: ❗ Expected validation error for bad 'videoTime', but got 200\nE       assert 200 >= 400\nE        +  where 200 = <Response [200]>.status_code\n\ntest_api_event.py:55: AssertionError"}, "start": 1753984503004, "stop": 1753984503128, "uuid": "a7807250-7456-48f9-93ea-5fdfbae81ced", "historyId": "abbcd9dd0e5c032cfcef704a23c6e32a", "testCaseId": "abbcd9dd0e5c032cfcef704a23c6e32a", "fullName": "test.test_api_event.TestApiEvent#test_post_invalid_video_time", "labels": [{"name": "epic", "value": "Backend API Validation"}, {"name": "feature", "value": "POST /api/event"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_api_event"}, {"name": "subSuite", "value": "TestApiEvent"}, {"name": "host", "value": "EladSalti"}, {"name": "thread", "value": "59628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_api_event"}], "titlePath": ["test", "test_api_event.py", "TestApiEvent"]}